Automattic/newspack-newsletters
# Use MJML in the browser

### All Submissions:






### Changes proposed in this Pull Request:



Instead of calling the MJML API, convert the post content into email-compliant HTML using the user's browser.

Closes #400.

### How to test the changes in this Pull Request:

1. On a site where the plugin has been set up before, everything should work normally - test by creating newsletters, sending test emails, sending campaign.
2. Observe there are no fields for MJML credentials in the settings.
2. On a site which did not yet have the plugin installed, observe the initial modal (when trying to create a newsletter before filling out the settings) does not require setting anything MJML-related 

### Other information:
